ledger/participant-integration-api/src/main/scala/platform/store/backend/VerifiedDataSource.scala

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.backend
+
+import com.daml.logging.{ContextualizedLogger, LoggingContext}
+import com.daml.platform.store.DbType
+import com.daml.timer.RetryStrategy
+import javax.sql.DataSource
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.duration.DurationInt
+import scala.util.Using
+
+object VerifiedDataSource {
```

Comment:
There are many classes in `com.daml.platform.store.backend`. At least for me, a short documentation of each class helps me navigating the code base.

```suggestion
/** Returns a DataSource that is guaranteed to be connected to a responsive, compatible database. */
object VerifiedDataSource {
```